docs/reference/mpremote.rst

```diff
@@ -249,38 +476,198 @@ Examples
 
   mpremote
 
+This automatically connects to the first available device and implicitly runs
+the ``repl`` command.
+
+.. code-block:: bash
+
   mpremote a1
 
-  mpremote connect /dev/ttyUSB0 repl
+This connects to the device at ``/dev/ttyACM1`` and implicitly runs the ``repl``
+command.
 
-  mpremote ls
+.. code-block:: bash
+
+  mpremote connect /dev/ttyUSB0
+
+This allows you to explicitly specify which device to connect to, and as above,
+implicitly runs the ``repl`` command.
+
+.. code-block:: bash
 
   mpremote a1 ls
 
+This connects to the device at ``/dev/ttyACM0`` and then runs the ``ls`` command.
```

Comment:
Consider using `c0` in one of these examples to reiterate that `mpremote` works on Windows?